Sources/SwiftDocC/Infrastructure/Link Resolution/PathHierarchyBasedLinkResolver.swift

```diff
@@ -58,6 +58,26 @@ final class PathHierarchyBasedLinkResolver {
         }
     }
     
+    /// Traverse all symbols of the same kind that have collisions.
+    func traverseOverloadedSymbols(_ observe: (_ overloadedSymbols: [ResolvedTopicReference]) -> Void) {
+        let overloadableKinds = ["method", "type.method", "func", "init", "macro", "subscript", "func.op"]
```

Comment:
Is there a better place or way to define this list? For example in 'SymbolGraph.Symbol.KinIdentifier'? But I recall @bitjammer talking about how the symbol graph format should not be tied to a specified programming language so maybe somewhere in DocC's source code would be better?

Anyway this list seems important enough it should be public and defined in some more obvious place.

Reply:
I was just thinking about this :D
Somewhere in DocC sounds right to me. I'm not sure the best place to put it though--maybe in the PathHierarchy, since that's currently where DocC determines symbol name collisions?